src/pages/home/category-item/category-item.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { useStyles } from './category-item.style';
+
+const Categories = ({ categoryName, categoryImage, categoryCode }) => {
+  const props = {
```

Comment:
Please, remove redundant variable `props`, just put actual object inside `useStyles`

Reply:
It is resolved here https://github.com/horondi/horondi_client_fe/pull/54